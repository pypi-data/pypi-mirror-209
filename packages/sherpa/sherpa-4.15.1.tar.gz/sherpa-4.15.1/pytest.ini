[pytest]
addopts = -rs --ignore=setup.py --ignore=test_requirements.txt
norecursedirs = .git build dist tmp* .eggs
text_file_format = rst
doctest_plus = enabled
doctest_plus_atol = 1e-4
doctest_optionflags =
    NORMALIZE_WHITESPACE
    ELLIPSIS
    FLOAT_CMP
doctest_norecursedirs =
    extern
    sherpa-test-data
    scripts
    # Blacklist everything and then remove from this list when it's ready
    docs/_examples
    docs/code
    docs/data
    docs/developer
    docs/examples
    docs/extra
    docs/fit
    docs/install.rst
    docs/mcmc
    docs/model_classes
    docs/models
    docs/optimisers
    docs/overview
    docs/plots
    docs/statistics
    docs/ui
    sherpa/__init__.py
    sherpa/astro
    sherpa/conftest.py
    sherpa/estmethods
    sherpa/fit.py
    sherpa/image
    sherpa/include
    sherpa/io.py
    sherpa/models/model.py
    sherpa/models/parameter.py
    sherpa/models/regrid.py
    sherpa/optmethods
    sherpa/plot
    sherpa/sim
    sherpa/static
    sherpa/stats
    sherpa/tests
    sherpa/ui
    sherpa/utils

doctest_subpackage_requires =
    sherpa/astro/io/crates_backend.py = pycrates
    sherpa/astro/io/pyfits_backend.py = astropy
    sherpa/plot/pylab_backend.py = matplotlib

