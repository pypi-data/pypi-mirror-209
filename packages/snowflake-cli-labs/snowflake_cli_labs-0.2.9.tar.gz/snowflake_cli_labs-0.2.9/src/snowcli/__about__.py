from __future__ import annotations

VERSION = "0.2.9"
