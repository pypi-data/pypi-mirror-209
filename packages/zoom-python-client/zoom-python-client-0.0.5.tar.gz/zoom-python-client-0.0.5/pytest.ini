[pytest]
; exclude unrelated folders
norecursedirs = .* .git env venv docs logs example dist
addopts = --cov-report term-missing --cov zoom_python_client