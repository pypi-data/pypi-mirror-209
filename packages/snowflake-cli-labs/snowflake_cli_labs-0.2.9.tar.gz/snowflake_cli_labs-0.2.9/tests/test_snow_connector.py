import textwrap
from unittest import mock

import pytest
from snowcli.snow_connector import SnowflakeConnector


# Used as a solution to syrupy having some problems with comparing multilines string
class custom_str(str):
    def __repr__(self):
        return str(self)


@mock.patch("snowflake.connector")
def test_createFunction(_, snapshot):
    connector = SnowflakeConnector(
        connection_name="foo", snowsql_config=mock.MagicMock()
    )
    connector.ctx.execute_stream.return_value = (None, None)

    connector.createFunction(
        name="nameValue",
        inputParameters="(string a, variant b)",
        returnType="returnTypeValue",
        handler="handlerValue",
        imports="import1, import2",
        database="databaseValue",
        schema="schemaValue",
        role="roleValue",
        warehouse="warehouseValue",
        overwrite=True,
        packages=["aaa", "bbb"],
    )
    query_io, *_ = connector.ctx.execute_stream.call_args.args
    query_str = query_io.getvalue()
    print(custom_str(query_str))
    assert custom_str(query_str) == snapshot


@mock.patch("snowflake.connector")
def test_createProcedure(_, snapshot):
    connector = SnowflakeConnector(
        connection_name="foo", snowsql_config=mock.MagicMock()
    )
    connector.ctx.execute_stream.return_value = (None, None)

    connector.createProcedure(
        name="nameValue",
        inputParameters="(string a, variant b)",
        returnType="returnTypeValue",
        handler="handlerValue",
        imports="import1, import2",
        database="databaseValue",
        schema="schemaValue",
        role="roleValue",
        warehouse="warehouseValue",
        overwrite=True,
        packages=["aaa", "bbb"],
        execute_as_caller=True,
    )
    query, *_ = connector.ctx.execute_stream.call_args.args
    assert query.getvalue() == snapshot


@mock.patch("snowflake.connector")
def test_executeFunction(_, snapshot):
    connector = SnowflakeConnector(
        connection_name="foo", snowsql_config=mock.MagicMock()
    )
    connector.ctx.execute_stream.return_value = (None, None)

    connector.executeFunction(
        function="functionValue",
        database="databaseValue",
        schema="schemaValue",
        role="roleValue",
        warehouse="warehouseValue",
    )
    query, *_ = connector.ctx.execute_stream.call_args.args
    assert query.getvalue() == snapshot


@mock.patch("snowflake.connector")
def test_executeProcedure(_, snapshot):
    connector = SnowflakeConnector(
        connection_name="foo", snowsql_config=mock.MagicMock()
    )
    connector.ctx.execute_stream.return_value = (None, None)

    connector.executeProcedure(
        procedure="procedureValue",
        database="databaseValue",
        schema="schemaValue",
        role="roleValue",
        warehouse="warehouseValue",
    )
    query, *_ = connector.ctx.execute_stream.call_args.args
    assert query.getvalue() == snapshot


@mock.patch("snowflake.connector")
def test_describeFunction(_, snapshot):
    connector = SnowflakeConnector(
        connection_name="foo", snowsql_config=mock.MagicMock()
    )
    connector.ctx.execute_stream.return_value = (None, None)

    connector.describeFunction(
        database="databaseValue",
        schema="schemaValue",
        role="roleValue",
        warehouse="warehouseValue",
        signature="signatureValue",
        name="nameValue",
        inputParameters="(string a, variant b)",
        show_exceptions="show_exceptionsValue",
    )
    query, *_ = connector.ctx.execute_stream.call_args.args
    assert query.getvalue() == snapshot


@mock.patch("snowflake.connector")
def test_describeProcedure(_, snapshot):
    connector = SnowflakeConnector(
        connection_name="foo", snowsql_config=mock.MagicMock()
    )
    connector.ctx.execute_stream.return_value = (None, None)

    connector.describeProcedure(
        database="databaseValue",
        schema="schemaValue",
        role="roleValue",
        warehouse="warehouseValue",
        signature="signatureValue",
        name="nameValue",
        inputParameters="(string a, variant b)",
        show_exceptions="show_exceptionsValue",
    )
    query, *_ = connector.ctx.execute_stream.call_args.args
    assert query.getvalue() == snapshot


@mock.patch("snowflake.connector")
def test_listFunctions(_, snapshot):
    connector = SnowflakeConnector(
        connection_name="foo", snowsql_config=mock.MagicMock()
    )
    connector.ctx.execute_stream.return_value = (None, None)

    connector.listFunctions(
        database="databaseValue",
        schema="schemaValue",
        role="roleValue",
        warehouse="warehouseValue",
        like="likeValue",
    )
    query, *_ = connector.ctx.execute_stream.call_args.args
    assert query.getvalue() == snapshot


@mock.patch("snowflake.connector")
def test_listStages(_, snapshot):
    connector = SnowflakeConnector(
        connection_name="foo", snowsql_config=mock.MagicMock()
    )
    connector.ctx.execute_stream.return_value = (None, None)

    connector.listStages(
        database="databaseValue",
        schema="schemaValue",
        role="roleValue",
        warehouse="warehouseValue",
        like="likeValue",
    )
    query, *_ = connector.ctx.execute_stream.call_args.args
    assert query.getvalue() == snapshot


@pytest.mark.parametrize(
    "stage_name", [("namedStageValue"), ("snow://embeddedStageValue")]
)
@mock.patch("snowflake.connector")
def test_listStage(_, snapshot, stage_name):
    connector = SnowflakeConnector(
        connection_name="foo", snowsql_config=mock.MagicMock()
    )
    connector.ctx.execute_stream.return_value = (None, None)

    connector.listStage(
        database="databaseValue",
        schema="schemaValue",
        role="roleValue",
        warehouse="warehouseValue",
        name=stage_name,
        like="likeValue",
    )
    query, *_ = connector.ctx.execute_stream.call_args.args
    assert query.getvalue() == snapshot


@pytest.mark.parametrize(
    "stage_name", [("namedStageValue"), ("snow://embeddedStageValue")]
)
@mock.patch("snowflake.connector")
def test_getStage(_, snapshot, stage_name):
    connector = SnowflakeConnector(
        connection_name="foo", snowsql_config=mock.MagicMock()
    )
    connector.ctx.execute_stream.return_value = (None, None)

    connector.getStage(
        database="databaseValue",
        schema="schemaValue",
        role="roleValue",
        warehouse="warehouseValue",
        name=stage_name,
        path="pathValue",
    )
    query, *_ = connector.ctx.execute_stream.call_args.args
    assert query.getvalue() == snapshot


@mock.patch("snowflake.connector")
def test_setProcedureComment(_, snapshot):
    connector = SnowflakeConnector(
        connection_name="foo", snowsql_config=mock.MagicMock()
    )
    connector.ctx.execute_stream.return_value = (None, None)

    connector.setProcedureComment(
        database="databaseValue",
        schema="schemaValue",
        role="roleValue",
        warehouse="warehouseValue",
        signature="signatureValue",
        name="nameValue",
        inputParameters="(string a, variant b)",
        show_exceptions="show_exceptionsValue",
        comment="commentValue",
    )
    query, *_ = connector.ctx.execute_stream.call_args.args
    assert query.getvalue() == snapshot


@pytest.mark.parametrize(
    "stage_name", [("namedStageValue"), ("snow://embeddedStageValue")]
)
@mock.patch("snowflake.connector")
def test_putStage(_, snapshot, stage_name):
    connector = SnowflakeConnector(
        connection_name="foo", snowsql_config=mock.MagicMock()
    )
    connector.ctx.execute_stream.return_value = (None, None)

    connector.putStage(
        database="databaseValue",
        schema="schemaValue",
        role="roleValue",
        warehouse="warehouseValue",
        name=stage_name,
        path="pathValue",
        overwrite=True,
        parallel="parallelValue",
    )
    query, *_ = connector.ctx.execute_stream.call_args.args
    assert textwrap.dedent(query.getvalue()) == snapshot


@pytest.mark.parametrize(
    "stage_name", [("namedStageValue"), ("snow://embeddedStageValue")]
)
@mock.patch("snowflake.connector")
def test_removeFromStage(_, snapshot, stage_name):
    connector = SnowflakeConnector(
        connection_name="foo", snowsql_config=mock.MagicMock()
    )
    connector.ctx.execute_stream.return_value = (None, None)

    connector.removeFromStage(
        database="databaseValue",
        schema="schemaValue",
        role="roleValue",
        warehouse="warehouseValue",
        name=stage_name,
        path="/pathValue",
    )
    query, *_ = connector.ctx.execute_stream.call_args.args
    assert query.getvalue() == snapshot


@mock.patch("snowflake.connector")
def test_createStage(_, snapshot):
    connector = SnowflakeConnector(
        connection_name="foo", snowsql_config=mock.MagicMock()
    )
    connector.ctx.execute_stream.return_value = (None, None)

    connector.createStage(
        database="databaseValue",
        schema="schemaValue",
        role="roleValue",
        warehouse="warehouseValue",
        name="nameValue",
    )
    query, *_ = connector.ctx.execute_stream.call_args.args
    assert query.getvalue() == snapshot


@mock.patch("snowflake.connector")
def test_dropStage(_, snapshot):
    connector = SnowflakeConnector(
        connection_name="foo", snowsql_config=mock.MagicMock()
    )
    connector.ctx.execute_stream.return_value = (None, None)

    connector.dropStage(
        database="databaseValue",
        schema="schemaValue",
        role="roleValue",
        warehouse="warehouseValue",
        name="nameValue",
    )
    query, *_ = connector.ctx.execute_stream.call_args.args
    assert query.getvalue() == snapshot


@mock.patch("snowflake.connector")
def test_listProcedures(_, snapshot):
    connector = SnowflakeConnector(
        connection_name="foo", snowsql_config=mock.MagicMock()
    )
    connector.ctx.execute_stream.return_value = (None, None)

    connector.listProcedures(
        database="databaseValue",
        schema="schemaValue",
        role="roleValue",
        warehouse="warehouseValue",
        like="likeValue",
    )
    query, *_ = connector.ctx.execute_stream.call_args.args
    assert query.getvalue() == snapshot


@mock.patch("snowflake.connector")
def test_dropFunction(_, snapshot):
    connector = SnowflakeConnector(
        connection_name="foo", snowsql_config=mock.MagicMock()
    )
    connector.ctx.execute_stream.return_value = (None, None)

    connector.dropFunction(
        database="databaseValue",
        schema="schemaValue",
        role="roleValue",
        warehouse="warehouseValue",
        signature="signatureValue",
    )
    query, *_ = connector.ctx.execute_stream.call_args.args
    assert query.getvalue() == snapshot


@mock.patch("snowflake.connector")
def test_dropProcedure(_, snapshot):
    connector = SnowflakeConnector(
        connection_name="foo", snowsql_config=mock.MagicMock()
    )
    connector.ctx.execute_stream.return_value = (None, None)

    connector.dropProcedure(
        database="databaseValue",
        schema="schemaValue",
        role="roleValue",
        warehouse="warehouseValue",
        signature="signatureValue",
    )
    query, *_ = connector.ctx.execute_stream.call_args.args
    assert query.getvalue() == snapshot


@mock.patch("snowflake.connector")
def test_listStreamlits(_, snapshot):
    connector = SnowflakeConnector(
        connection_name="foo", snowsql_config=mock.MagicMock()
    )
    connector.ctx.execute_stream.return_value = (None, None)

    connector.listStreamlits(
        database="databaseValue",
        schema="schemaValue",
        role="roleValue",
        warehouse="warehouseValue",
    )
    query, *_ = connector.ctx.execute_stream.call_args.args
    assert query.getvalue() == snapshot


@mock.patch("snowflake.connector")
def test_showWarehouses(_, snapshot):
    connector = SnowflakeConnector(
        connection_name="foo", snowsql_config=mock.MagicMock()
    )
    connector.ctx.execute_stream.return_value = (None, None)

    connector.showWarehouses(
        database="databaseValue",
        schema="schemaValue",
        role="roleValue",
        warehouse="warehouseValue",
        like="likeValue",
    )
    query, *_ = connector.ctx.execute_stream.call_args.args
    assert query.getvalue() == snapshot


@mock.patch("snowflake.connector")
def test_createStreamlit(_, snapshot):
    connector = SnowflakeConnector(
        connection_name="foo", snowsql_config=mock.MagicMock()
    )
    connector.ctx.execute_stream.return_value = (None, None)

    connector.createStreamlit(
        database="databaseValue",
        schema="schemaValue",
        role="roleValue",
        warehouse="warehouseValue",
        name="nameValue",
        file="fileValue",
    )
    query, *_ = connector.ctx.execute_stream.call_args.args
    assert query.getvalue() == snapshot


@mock.patch("snowflake.connector")
def test_createStreamlitFromStage(_, snapshot):
    connector = SnowflakeConnector(
        connection_name="foo", snowsql_config=mock.MagicMock()
    )
    connector.ctx.execute_stream.return_value = (None, None)

    connector.createStreamlit(
        database="databaseValue",
        schema="schemaValue",
        role="roleValue",
        warehouse="warehouseValue",
        name="nameValue",
        file="fileValue",
        from_stage_command="FROM @stageValue",
    )
    query, *_ = connector.ctx.execute_stream.call_args.args
    assert query.getvalue() == snapshot


@mock.patch("snowflake.connector")
def test_shareStreamlit(_, snapshot):
    connector = SnowflakeConnector(
        connection_name="foo", snowsql_config=mock.MagicMock()
    )
    connector.ctx.execute_stream.return_value = (None, None)

    connector.shareStreamlit(
        database="databaseValue",
        schema="schemaValue",
        role="roleValue",
        warehouse="warehouseValue",
        name="nameValue",
        to_role="to_roleValue",
    )
    query, *_ = connector.ctx.execute_stream.call_args.args
    assert query.getvalue() == snapshot


@mock.patch("snowflake.connector")
def test_dropStreamlit(_, snapshot):
    connector = SnowflakeConnector(
        connection_name="foo", snowsql_config=mock.MagicMock()
    )
    connector.ctx.execute_stream.return_value = (None, None)

    connector.dropStreamlit(
        database="databaseValue",
        schema="schemaValue",
        role="roleValue",
        warehouse="warehouseValue",
        name="nameValue",
    )
    query, *_ = connector.ctx.execute_stream.call_args.args
    assert query.getvalue() == snapshot


@mock.patch("snowflake.connector")
def test_deployStreamlit(_, snapshot):
    connector = SnowflakeConnector(
        connection_name="foo", snowsql_config=mock.MagicMock()
    )
    connector.ctx.execute_stream.return_value = (mock.MagicMock(),)
    connector.uploadFileToStage = mock.MagicMock()

    connector.deployStreamlit(
        name="nameValue",
        file_path="file_pathValue",
        stage_path="stage_pathValue",
        role="roleValue",
        database="databaseValue",
        schema="schemaValue",
        overwrite=True,
    )
    query, *_ = connector.ctx.execute_stream.call_args.args
    assert query.getvalue() == snapshot


@mock.patch("snowflake.connector")
def test_describeStreamlit(_, snapshot):
    connector = SnowflakeConnector(
        connection_name="foo", snowsql_config=mock.MagicMock()
    )
    connector.ctx.execute_stream.return_value = (None, None)

    connector.describeStreamlit(
        name="nameValue",
        database="databaseValue",
        schema="schemaValue",
        role="roleValue",
        warehouse="warehouseValue",
    )
    query, *_ = connector.ctx.execute_stream.call_args.args
    assert query.getvalue() == snapshot
