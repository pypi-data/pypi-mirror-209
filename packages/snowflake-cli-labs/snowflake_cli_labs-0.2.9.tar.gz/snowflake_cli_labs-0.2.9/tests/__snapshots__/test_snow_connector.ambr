# serializer version: 1
# name: test_createFunction
  '''
  use database databaseValue;
  use schema schemaValue;
  use warehouse warehouseValue;
  CREATE OR REPLACE  FUNCTION nameValue(string a, variant b)
           RETURNS returnTypeValue
           LANGUAGE PYTHON
           RUNTIME_VERSION=3.8
           IMPORTS=('import1, import2')
           HANDLER='handlerValue'
           PACKAGES=('aaa','bbb');
  describe function nameValue(a, b);
  '''
# ---
# name: test_createProcedure
  '''
  use database databaseValue;
  use schema schemaValue;
  use warehouse warehouseValue;
  
  CREATE OR REPLACE  PROCEDURE nameValue(string a, variant b)
           RETURNS returnTypeValue
           LANGUAGE PYTHON
           RUNTIME_VERSION=3.8
           IMPORTS=('import1, import2')
           HANDLER='handlerValue'
           PACKAGES=('aaa','bbb')
           EXECUTE AS CALLER;
  
  
  describe PROCEDURE nameValue(a, b);
  '''
# ---
# name: test_createStage
  '''
  use role roleValue;
  use warehouse warehouseValue;
  use database databaseValue;
  use schema schemaValue;
  
  create stage if not exists nameValue;
  '''
# ---
# name: test_createStreamlit
  '''
  use role roleValue;
  use database databaseValue;
  use schema schemaValue;
  use warehouse warehouseValue;
  
  create streamlit nameValue
    
    MAIN_FILE = 'fileValue'
    QUERY_WAREHOUSE = warehouseValue;
  
  show streamlits;
  describe streamlit nameValue;
  
  alter streamlit nameValue checkout;
  '''
# ---
# name: test_createStreamlitFromStage
  '''
  use role roleValue;
  use database databaseValue;
  use schema schemaValue;
  use warehouse warehouseValue;
  
  create streamlit nameValue
    FROM @stageValue
    MAIN_FILE = 'fileValue'
    QUERY_WAREHOUSE = warehouseValue;
  
  show streamlits;
  describe streamlit nameValue;
  
  alter streamlit nameValue checkout;
  '''
# ---
# name: test_deployStreamlit
  '''
  use database databaseValue;
  use schema schemaValue;
  use role roleValue;
  
  CALL SYSTEM$GENERATE_STREAMLIT_URL_FROM_NAME('nameValue');
  '''
# ---
# name: test_describeFunction
  '''
  use role roleValue;
  use warehouse warehouseValue;
  use database databaseValue;
  use schema schemaValue;
  desc FUNCTION signatureValue;
  '''
# ---
# name: test_describeProcedure
  '''
  use role roleValue;
  use warehouse warehouseValue;
  use database databaseValue;
  use schema schemaValue;
  desc PROCEDURE signatureValue;
  '''
# ---
# name: test_describeStreamlit
  '''
  use database databaseValue;
  use schema schemaValue;
  use role roleValue;
  
  CALL SYSTEM$GENERATE_STREAMLIT_URL_FROM_NAME('nameValue');
  '''
# ---
# name: test_dropFunction
  '''
  use role roleValue;
  use warehouse warehouseValue;
  use database databaseValue;
  use schema schemaValue;
  drop function signatureValue;
  '''
# ---
# name: test_dropProcedure
  '''
  use role roleValue;
  use warehouse warehouseValue;
  use database databaseValue;
  use schema schemaValue;
  drop procedure signatureValue;
  '''
# ---
# name: test_dropStage
  '''
  use role roleValue;
  use warehouse warehouseValue;
  use database databaseValue;
  use schema schemaValue;
  
  drop stage nameValue;
  '''
# ---
# name: test_dropStreamlit
  '''
  use role roleValue;
  use database databaseValue;
  use schema schemaValue;
  use warehouse warehouseValue;
  drop streamlit "nameValue";
  '''
# ---
# name: test_executeFunction
  '''
  use role roleValue;
  use warehouse warehouseValue;
  use database databaseValue;
  use schema schemaValue;
  select functionValue;
  '''
# ---
# name: test_executeProcedure
  '''
  use role roleValue;
  use warehouse warehouseValue;
  use database databaseValue;
  use schema schemaValue;
  call procedureValue;
  '''
# ---
# name: test_getStage[namedStageValue]
  '''
  use role roleValue;
  use warehouse warehouseValue;
  use database databaseValue;
  use schema schemaValue;
  
  get @namedStageValue file://pathValue/
  '''
# ---
# name: test_getStage[snow://embeddedStageValue]
  '''
  use role roleValue;
  use warehouse warehouseValue;
  use database databaseValue;
  use schema schemaValue;
  
  get snow://embeddedStageValue file://pathValue/
  '''
# ---
# name: test_listFunctions
  '''
  use role roleValue;
  use warehouse warehouseValue;
  use database databaseValue;
  use schema schemaValue;
  show user functions like 'likeValue';
  select "name", "created_on", "arguments", "language" from table(result_scan(last_query_id()));
  '''
# ---
# name: test_listProcedures
  '''
  use role roleValue;
  use warehouse warehouseValue;
  use database databaseValue;
  use schema schemaValue;
  show user procedures like 'likeValue';
  select "name", "created_on", "arguments" from table(result_scan(last_query_id()));
  '''
# ---
# name: test_listStage[namedStageValue]
  '''
  use role roleValue;
  use warehouse warehouseValue;
  use database databaseValue;
  use schema schemaValue;
  ls @namedStageValue;
  '''
# ---
# name: test_listStage[snow://embeddedStageValue]
  '''
  use role roleValue;
  use warehouse warehouseValue;
  use database databaseValue;
  use schema schemaValue;
  ls snow://embeddedStageValue;
  '''
# ---
# name: test_listStages
  '''
  use role roleValue;
  use warehouse warehouseValue;
  use database databaseValue;
  use schema schemaValue;
  show stages;
  '''
# ---
# name: test_listStreamlits
  '''
  use role roleValue;
  use warehouse warehouseValue;
  use database databaseValue;
  use schema schemaValue;
  show streamlits;
  '''
# ---
# name: test_putStage[namedStageValue]
  '''
  use role roleValue;
  use warehouse warehouseValue;
  use database databaseValue;
  use schema schemaValue;
  
  put file://pathValue @namedStageValue auto_compress=false parallel=parallelValue overwrite=True;
  '''
# ---
# name: test_putStage[snow://embeddedStageValue]
  '''
  use role roleValue;
  use warehouse warehouseValue;
  use database databaseValue;
  use schema schemaValue;
  
  put file://pathValue snow://embeddedStageValue auto_compress=false parallel=parallelValue overwrite=True;
  '''
# ---
# name: test_removeFromStage[namedStageValue]
  '''
  use role roleValue;
  use warehouse warehouseValue;
  use database databaseValue;
  use schema schemaValue;
  
  remove @namedStageValue/pathValue
  '''
# ---
# name: test_removeFromStage[snow://embeddedStageValue]
  '''
  use role roleValue;
  use warehouse warehouseValue;
  use database databaseValue;
  use schema schemaValue;
  
  remove snow://embeddedStageValue/pathValue
  '''
# ---
# name: test_setProcedureComment
  '''
  use role roleValue;
  use warehouse warehouseValue;
  use database databaseValue;
  use schema schemaValue;
  alter PROCEDURE signatureValue SET COMMENT = $$commentValue$$;
  '''
# ---
# name: test_shareStreamlit
  '''
  use role roleValue;
  use warehouse warehouseValue;
  use database databaseValue;
  use schema schemaValue;
  
  grant usage on streamlit nameValue to role to_roleValue;
  '''
# ---
# name: test_showWarehouses
  '''
  use role roleValue;
  use warehouse warehouseValue;
  use database databaseValue;
  use schema schemaValue;
  show warehouses;
  '''
# ---
